app/src/main/java/org/p2p/wallet/claim/api/request/GetClaimFeeRpcRequest.kt

```diff
@@ -21,5 +21,5 @@ class GetClaimFeeRpcRequest(
         put("amount", amount)
     }
 ) {
-    override val typeOfResult: Type = BundleFeeResponse::class.java
+    override val typeOfResult: Type = FeesResponse::class.java
```

Comment:
there are a lot of Fees in our app
i would add some context
ClaimFeeResponse for exampe